packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1458,4 +1458,40 @@ void main() {
       ' possible elevation values.'
     ));
   });
+
+  testWidgets('ExpansionPanel.panelColor test', (WidgetTester tester) async {
```

Comment:
We should add another similar test for `ExpansionPanelRadio`, just to make sure we don't regress there.